codesign/lib/src/google_cloud_storage.dart

```diff
@@ -1,49 +0,0 @@
-// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Should this file be removed rather than just deleting the content of it?

Reply:
umm for some reasons this file is shown as removed on github ui on my end. 
<img width="319" alt="Screen Shot 2022-11-03 at 2 41 11 PM" src="https://user-images.githubusercontent.com/38773539/199839992-8162ac20-3a98-486c-b651-022db83c6841.png">